src/components/People/People.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Hero from '../HeroCard/Hero';
+import IHero from '../../interfaces/hero';
+import './people.css';
+
+interface PeopleProps {
+  people: IHero[];
+  searchItem: string;
+}
+
+export default class People extends React.Component<PeopleProps> {
+  constructor(props: PeopleProps) {
+    super(props);
+  }
+
+  render() {
+    const { people, searchItem } = this.props;
+    const filteredPeople = people.filter((item) =>
+      item.name.includes(searchItem)
+    );
+    return (
+      <div className="cards-container">
+        {filteredPeople.map((item) => (
+          <Hero
+            key={item.url}
```

Comment:
Why you create so many props for component. Better use IHero. <Hero hero={item} key={item.url}/>